content/pages/404.md

```diff
@@ -22,7 +22,7 @@ permalink: false
           <p>
             Try the search box or one of the common questions below.
           </p>
-          <div class="call-out">
+          <div class="feature">
```

Comment:
@tiffanybbrown-va this broke the layout on the 404 page a bit - I'm seeing a previously hidden "Search:" label, and the button is off-center

<img width="738" alt="vets_gov__vets_gov" src="https://cloud.githubusercontent.com/assets/4106356/24314490/1981e2e4-109f-11e7-97c2-08e495336f06.png">